### PR TITLE
Add warmup for logits processors

### PR DESCRIPTION
**Pull request by kdamaszk**
# What does this PR do?

Add warmup for logits processors. Do not slice input_ids to ensure static shapes.

### CODE REVIEW

**Comment by kdamaszk** [comment]
@regisss please review

**Review by regisss — APPROVED** [review]
LGTM!
